sites/public/pages/eligibility/income.tsx

```diff
@@ -49,13 +49,16 @@ const EligibilityIncome = () => {
   }
 
   function getFilterUrl() {
-    const params: ListingFilterState = {}
+    const state: ListingFilterState = {}
 
     if (eligibilityRequirements.age < SENIOR_AGE) {
-      params.seniorHousing = false
+      state.seniorHousing = false
+    }
+    if (eligibilityRequirements.disability === DISABILITY_NO) {
+      state.specialNeedsHousing = false
     }
```

Comment:
What about the true case?

Reply:
We don't need to do any filtering if this is true or if the user chooses not to disclose their status. We will display communities designated for the accessibility community as well as other listings. 

The only action we need to take is to filter out these accessibility communities if the user does not have a disability that qualifies them for that community type.

Reply:
It was my impression that users would want to be able to filter down to only the reserved communities, which is not possible with this implementation. If the user chose not to disclose, they would select the "prefer not to say", which would not apply any filtering. It seems counter-intuitive to me that declaring their status has no effect on the filters.

Reply:
They can filter for the community types using the filter modal. 

If someone says they have a disability, they also qualify for housing that doesn't require you to have a disability. Why would we hide all of these other results if they qualify for them?

Reply:
Ah! I thought the question was more along the lines of "Do you want to see homes reserved for people with disabilities?"

Reply:
No the eligibility questionnaire is meant to show you everything you're eligible for.

Reply:
Ah, I was mistaken in how I was thinking about it, thanks!